ethers-etherscan/src/utils.rs

```diff
@@ -3,8 +3,7 @@ use ethers_core::types::Address;
 use semver::Version;
 use serde::{Deserialize, Deserializer};
 
-static SOLC_BIN_LIST_URL: &str =
-    "https://raw.githubusercontent.com/ethereum/solc-bin/gh-pages/bin/list.txt";
+static SOLC_BIN_LIST_URL: &str = "https://binaries.soliditylang.org/bin/list.txt";
```

Comment:
[ethereum/solc-bin](https://github.com/ethereum/solc-bin):
> The content is mirrored to https://binaries.soliditylang.org/ where it can be easily downloaded over HTTPS without any authentication, rate limiting or the need to use git.

Reply:
#2198